atd-vze/src/views/Crashes/Recommendations/Recommendations.js

```diff
@@ -68,6 +68,12 @@ const Recommendations = ({ crashId }) => {
       .catch(error => console.error(error));
   };
 
+  // Function to alphabetize an array by a certain field
+  const alphabetizeArray = (array, field) => {
+    array.sort((a, b) => a[field].localeCompare(b[field]));
+    return array;
+  };
+
```

Comment:
An alternative to sorting in the JavaScript could be to add an `order_by` clause to the `atd__coordination_partners_lkp` part of the `GET_RECOMMENDATIONS` query. This would serve up the ordered results directly to the front end. I'll let others weigh in, but I think that less code is a perk. 

You can try that out in the [Hasura console](https://vzd-staging.austinmobility.io/console/api-explorer) - GraphiQL is super helpful with building queries like this.

Reply:
Thanks for the feedback Mike! That's actually what I originally tried doing but it wasn't working so I switched gears, but I just attempted it again a little differently and got it to work! I'll push those changes because I agree, less code is a perk.

Reply:
Anytime - glad it worked out! 🎉

Reply:
If you're curious, what I was attempting to do the first go around was adding the `order_by` clause under the `recommendations` part of the  query, so I was getting an array of recommendations sorted alphabetically by their coordination partners rather than sorting the query results for `atd__coordination_partners_lkp` directly.